app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -294,8 +312,40 @@ class NimbusExperimentSerializer(
     population_percent = serializers.DecimalField(
         7, 4, min_value=0.0, max_value=100.0, required=False
     )
+    status = serializers.ChoiceField(
+        choices=NimbusExperiment.Status.choices,
+        required=False,
+        validators=[
+            NimbusStatusTransitionValidator(
+                transitions={
+                    NimbusExperiment.Status.DRAFT: (
+                        NimbusExperiment.Status.PREVIEW,
+                        NimbusExperiment.Status.REVIEW,
+                    ),
+                    NimbusExperiment.Status.PREVIEW: (
+                        NimbusExperiment.Status.DRAFT,
+                        NimbusExperiment.Status.REVIEW,
+                    ),
+                },
```

Comment:
Pls move these transitions over to the Nimbus constants file next to their respective enums cuz we'll have to update those together 🙏

Reply:
We have other related constant lists in here too - should they move also?

https://github.com/mozilla/experimenter/pull/4887/files#diff-da5192d8fdd27a14107b117c1f4fbfbbc6cbee34c61bc19dd273fc612781f7ebR204-R210